translations/translations.json

```diff
@@ -10158,7 +10158,13 @@
       "makeAppointment": "Termin vereinbaren",
       "appointmentNecessary": "Termin erforderlich",
       "makeAppointmentTooltipWithLink": "Vereinbare an diesem Tag einen Termin über <1>die Webseite</1> oder telefonisch.",
-      "theWebsite": "die Webseite"
+      "theWebsite": "die Webseite",
+      "nearYou_zero": "0 Standorte {{nearYou}}",
```

Comment:
:wrench: The naming here should be `Orte` instead of `Standorte`. Furthermore, using `In deiner Nähe` like this does sadly not really work as `In` is uppercase while it would need to be lowercase. Accordingly, please just duplicate the translation.

Reply:
I'm not using it here as a text. Its only for talkback to be heard.

Reply:
This might still alter how things are spoken. Furthermore, this might create completely grammatically incorrect sentences (due to word order, different translations, ...). Therefore we really don't want to combine translations in a hacky way like this.